interfaces/i18n.js

```diff
@@ -0,0 +1,14 @@
+type chrome$LanguageCode = string;
+
+type chrome$i18n = {
+  getAcceptLanguages(callback: (Array<chrome$LanguageCode>) => void): void,
+  getMessage(messageName: string, ...substitutions?: Array<*>): ?string,
+  getUILanguage(): string,
+  detectLanguage(text: string, callback: (result: {
+    isReliable: boolean,
+    languages: {
```

Comment:
`languages` is an array of these objects
https://developer.chrome.com/extensions/i18n#method-detectLanguage